### PR TITLE
COG-1453 Column Sorting available in the Filter User Lists

### PR DESCRIPTION
**Pull request by AlexanderSerbin**
#### Which User story does this relate to (link)?
https://osi-cwds.atlassian.net/browse/COG-1453

#### Brief feature description
Added custom sorting for Last Login column

### CODE REVIEW

File: app/javascript/views/groupUsersList/GroupUsersList.jsx [hunk 1]
```diff
@@ -23,7 +23,15 @@ class GroupUsersList extends PureComponent {
           columns={[
             { Header: 'Full Name', id: 'last_name', accessor: toFullName, minWidth: 200 },
             { Header: 'Status', id: 'enabled', accessor: accountStatusFormat, minWidth: 60 },
-            { Header: 'Last Login', id: 'last_login_date_time', minWidth: 150, accessor: lastLoginDate },
+            {
+              Header: 'Last Login',
```
Comment (Nasir2712) [inline app/javascript/views/groupUsersList/GroupUsersList.jsx]: is there a test case already written for this?

File: app/javascript/views/groupUsersList/GroupUsersList.jsx [hunk 2]
```diff
@@ -23,7 +23,15 @@ class GroupUsersList extends PureComponent {
...
+              id: 'last_login_date_time',
+              minWidth: 150,
+              accessor: lastLoginDate,
+              sortMethod: (a, b) => {
+                return new Date(b) - new Date(a)
```
Comment (kelfink) [inline app/javascript/views/groupUsersList/GroupUsersList.jsx]: The sortMethod in react-table is not well specified, but all the examples I have seen return either 1 or -1.  Is this working as intended?

Comment (kelfink) [reply app/javascript/views/groupUsersList/GroupUsersList.jsx]: Best documentation of the method is here: https://www.npmjs.com/package/react-table
So I'd suggest replacing the return with :
return (new Date(b) > new Date(a)) ? 1 : -1
```suggestion
                return new Date(b) - new Date(a)
```

File: app/javascript/views/groupUsersList/GroupUsersList.jsx (hunk 1 above)
Comment (AlexanderSerbin) [reply app/javascript/views/groupUsersList/GroupUsersList.jsx]: Added tests for date time comparator

File: app/javascript/views/groupUsersList/GroupUsersList.jsx (hunk 2 above)
Comment (AlexanderSerbin) [reply app/javascript/views/groupUsersList/GroupUsersList.jsx]: Done. Changed comparator function to "return new Date(a) > new Date(b) ? 1 : -1"

**Pull request merged**